### DISCUSSION TITLE
How to use the function `openpnm_to_im`

### DISCUSSION THREAD

**Discussion by MJL-JLU** [PMEAL/porespy#855] [Q&A]
I generated the PNM model in openpnm with the real structure, and now I want to export the model as im, but I haven't been successful in trying, how can I use this function?

**Comment by jgostick** [comment]
Can you be more specific about the problem you're having?

**Reply by MJL-JLU** [reply]
Dear Professor, thank you for your reply, these days I have not logged in, please forgive me not timely reply. 
I tried calling the openpnm_to_im function in porespy to test the effect. I generated a network in openpnm, but there was an error in calling this function, and there was no example of the function in the IO file, how to use this function or can you add some examplex to the function in the IO file?

**Reply by jgostick** [reply]
can you paste your error here?

**Reply by MJL-JLU** [reply]
im0=ps.io.openpnm_to_im(pn)

AttributeError: 'numpy.ndarray' object has no attribute 'io'
The network I create from the snow2 function
